jax/experimental/jax2tf/jax2tf.py

```diff
@@ -856,6 +895,7 @@ def pack(a, b):
 
     # Unpacks the first element of a tuple.
     def fst(t):
+      assert t.dtype == double_word_dtype
```

Comment:
nit: was this assert here for debugging purpose or do we actually want to keep it here?

Reply:
It acts as a documentation for the type of `t`, which matters because of the size of the shift.